package.json

```diff
@@ -2,12 +2,12 @@
   "name": "klicky",
   "version": "1.0.0",
   "description": "",
-  "main": "index.html",
+  "main": "test.html",
   "scripts": {
     "check": "node setup.js",
-    "start": "parcel index.html --open",
+    "start": "parcel test.html --open",
```

Comment:
let's keep these commands the same, to get to your test page, instead of running `npm run start` you should run `npm run start-test` with the original code